lib/Doctrine/ORM/Tools/Pagination/Paginator.php

```diff
@@ -102,9 +102,9 @@ public function count(): int
     /**
      * {@inheritdoc}
      *
-     * @psalm-return Traversable<array-key, T>
+     * @psalm-return Traversable<array-key, T>&ArrayIterator
```

Comment:
Here, we can't just use `ArrayIterator<array-key, T>` because that would make `T` appear in an invariant position, while it's declared as covariant.

Reply:
Do we want `ArrayIterator` to be part of the contract? I don't see a reason to change the return type.

Reply:
:thinking: hum you're right, it makes no sense to leak that info.